release/models/rib/openconfig-rib-bgp-tables.yang

```diff
@@ -311,6 +321,639 @@ submodule openconfig-rib-bgp-tables {
     }
   }
 
+  grouping l2vpn-evpn-loc-rib-top {
```

Comment:
Can you use the OC naming convention?

Reply:
Will update.

Reply:
I did a bit more investigation on this one also and the convention is to use the address-family without the `bgp-` prefix.  For example:

* https://github.com/openconfig/public/blob/master/release/models/rib/openconfig-rib-bgp.yang#L183
* https://github.com/openconfig/public/blob/master/release/models/rib/openconfig-rib-bgp.yang#L196